packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -340,11 +343,71 @@ def test_vip(dcos_api_session,
                 pm_app.purge(dcos_api_session)
     assert not errors
 
+    containers = list(marathon.Container) + [Container.POD]
+    return [(container, vip_net, proxy_net)
+            for container in containers
+            for vip_net in list(marathon.Network)
+            for proxy_net in list(marathon.Network)]
+
+
+@pytest.mark.parametrize('container', [marathon.Container.MESOS, Container.POD])
```

Comment:
This will be extended with DOCKER container, once the libnetwork support lands. Right?

Reply:
You got it. 
Adding libnetwork tests can be very handy, https://github.com/mainred/dcos/pull/1/files#diff-89643056102d7647724e7dc94be80843